Gemfile

```diff
@@ -37,7 +37,7 @@ group :development do
   gem "github_changelog_generator", '= 1.15.2',        require: false
 end
 group :system_tests do
-  gem "puppet_litmus", '<= 0.34.6', require: false, platforms: [:ruby]
+  gem "puppet_litmus", '< 1.0.0', require: false, platforms: [:ruby, :x64_mingw]
```

Comment:
why can't the 1.0.2 release be used?

Reply:
v0.36.2 and v1.0.2 both contain the fixed metadata script. But you're right, may as well just pull the 1.0.2 release :-)